app/graphql/types/query_type.rb

```diff
@@ -37,5 +40,46 @@ def news_item(id:)
     def tour(id:)
       Tour.find(id)
     end
+
+    # Provide contents from html files in `public/mobile-app/contents` through GraphQL query
+    #
+    # @param [String] name the file name
+    #
+    # @return [String] the contents of the file, if it exists - otherwise ""
+    def public_html_file(name:)
+      public_contents_folder = "#{Rails.root}/public/mobile-app/contents"
+      file_type = "html"
+      file = File.join(public_contents_folder, "#{name}.#{file_type}")
+
+      if query_file?(file, public_contents_folder, file_type)
+        return { content: File.read(file).squish }
+      end
+
+      { content: "" }
+    end
+
+    private
+
+      # Provide a list of files within a folder that are allowed to query through GraphQL
+      #
+      # @param [String] path the path to lookup
+      # @param [String] file_type the file type to lookup
+      #
+      # @return [Array] list of queryable files
+      def query_files_whitelist(path, file_type)
+        queryable_files = File.join(path, "*.#{file_type}")
+        Dir.glob(queryable_files)
+      end
+
+      # Is a file allowed to query through GraphQL or not?
```

Comment:
Richtiger Kommentar wäre: "Is a file allowed to be queried by the graphql server."

Reply:
done